core/src/main/scala/sttp/client3/RequestT.scala

```diff
@@ -233,6 +233,26 @@ case class RequestT[U[_], T, -R](
 
   def tag(k: String): Option[Any] = tags.get(k)
 
+  private val loggingOptionsTagKey = "loggingOptions"
+
+  /** Can only be applied when you use `LoggingBackend` */
+  def withLogSettings(
```

Comment:
I think I'd call this simply `.logSettings` as other methods don't have the `with` prefix